File: ClientJava/webservice/Personne.java

package webservice;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "personne", propOrder = {
    "admin",
    "login",
    "mdp",
    "nom",
    "prenom"
})
public class Personne {

    protected boolean admin;
    protected String login;
    protected String mdp;
    protected String nom;
    protected String prenom;

    
    public boolean isAdmin() {
        return admin;
    }

    
    public void setAdmin(boolean value) {
        this.admin = value;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String value) {
        this.login = value;
    }

    public String getMdp() {
        return mdp;
    }

  
    public void setMdp(String value) {
        this.mdp = value;
    }

   
    public String getNom() {
        return nom;
    }

    
    public void setNom(String value) {
        this.nom = value;
    }

    
    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String value) {
        this.prenom = value;
    }

}

File: Figaro/modele/domaine/Personne.php
<?php
	/**
	 * Classe métier représentant un user
	 */
	class Personne
	{
		public $id;
		public  $nom;
		public  $prenom;
		public  $login;
		public  $mdp;
		public  $admin;

		function __construct()
		{
			
		}
		public function setNom(string $n): void
		{
			$this->nom = $n;
		}
		public function setPrenom(string $p): void
		{
			$this->prenom = $p;
		}
		public function setLogin(string $l): void
		{
			$this->login = $l;
		}
		public function setAdmin(string $a): void
		{
			$this->admin = $a;
		}
		public function setMdp(string $m): void
		{
			$this->mdp = $m;
		}
	}
?>